rudder-web/src/main/scala/com/normation/rudder/appconfig/ConfigService.scala

```diff
@@ -356,6 +374,11 @@ class LDAPBasedConfigService(configFile: Config, repos: ConfigRepository, workfl
       SyslogTCP
     case _ => SyslogUDP
   }
+  private[this] implicit def toRelaySynchronisationMethod(p: RudderWebProperty): RelaySynchronizationMethod = p.value match {
+    case ClassicSynchronization.value  => ClassicSynchronization
+    case RsyncSynchronization.value    => RsyncSynchronization
+    case DisabledSynchronization.value => DisabledSynchronization
```

Comment:
Heren the match is not exhaustive. If the value is not recognized, it will fails. 
I thing we want to have "classic" as the default, so use:
```
  case RsyncSynchronization.value    => RsyncSynchronization
  case DisabledSynchronization.value => DisabledSynchronization
  case _                             => ClassicSynchronization
```

Reply:
you are totally right